config/Dockerfiles/centos7/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN yum -y install python-pip \
                     libvirt-devel virt-install file openssh mkisofs \
                     libvirt-client net-tools git python-krbV make \
                     libxslt-python krb5-workstation PyYAML python-ipaddress \
-                    python-requests jq buildah; \
+                    python-requests jq buildah git; \
```

Comment:
@14rcole Hi, looks like the repos are faulty, could you drop centos 7 change to Dockerfile and try the test. I'll fix this with rcm tools repo which has access to centos pkgs.
https://github.com/CentOS-PaaS-SIG/linchpin/blob/e40e92e0c1f2a1155c1090a20d5e3100246786d0/config/Dockerfiles/centos7/Dockerfile#L14-L15